tfx/orchestration/airflow/airflow_dag_runner_test.py

```diff
@@ -192,6 +210,70 @@ def testAirflowDagRunnerInitBackwardCompatible(self):
 
     self.assertEqual(airflow_config, runner._config.airflow_dag_config)
 
+  def testRuntimParam(self):
+    param = RuntimeParameter('name', str, 'tfx')
```

Comment:
Can you add a double-quote into this 'tfx' string to test quoting behavior?

Reply:
done, lmk if it looks good.